package.json

```diff
@@ -146,7 +146,7 @@
     "bignumber.js": "^9.1.2",
     "eventsource": "^2.0.2",
     "randombytes": "^2.1.0",
-    "stellar-base": "git+https://github.com/stellar/js-stellar-base#8d8b09b",
+    "stellar-base": "git+https://github.com/stellar/js-stellar-base#af1b94a",
```

Comment:
this is unrelated to renaming the publish destination, seems like it could have more impact, and some tests are failing, maybe related to this? maybe is worth a separate pr?